src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -36,6 +35,7 @@ def put(self, request, organization, alert_rule):
 
     def delete(self, request, organization, alert_rule):
         try:
+            # TODO: Ensure delete_alert_rule deletes triggers + actions as well.
```

Comment:
It should do since Django will follow references.

Reply:
That's what I thought & was hoping for. Just wanted to verify :-)